File: Entregable/Proyecto/BetizagastiGnocchi.BackEnd.Domain/Entities/Inspection.cs
using System;
using BetizagastiGnocchi.BackEnd.Domain.Enum;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace BetizagastiGnocchi.BackEnd.Domain.Entities
{
    public class Inspection
    {
		[Key]
		public int Id { get; set; }
		public InspectionStatus Status { get; set; }

        public DateTime DateAndTime { get; set; }

        public InspectionPlace Place { get; set; }

        public User Inspector { get; set; }
		public int VehicleId { get; set; }
		public virtual Vehicle Vehicle { get; set; }

		public virtual List<DamageRegistry> DamageRegistries { get; set; } = new List<DamageRegistry>();

    }
}

File: web/src/app/models/Inspection.ts
import { DamageRegistry } from "./DamageRegistry";
import { User } from "./User";
import { Vehicle } from "./Vehicle";
export class Inspection {
    DamageRegistries: DamageRegistry[]= new  Array<DamageRegistry>();
    Id: number;
    Status: number;
    DateAndTime: Date;
    Place: number;
    Inspector?: User;
    VehicleId: number;
    Vehicle: Vehicle;
}
